include/oboe/Definitions.h

```diff
@@ -834,6 +834,204 @@ namespace oboe {
         None = 3,
     };
 
+    /**
+     * Audio device type.
+     *
+     * Note that these match the device types defined in android/media/AudioDeviceInfo.java.
+     *
+     * Added in API level 36 for AAudio.
+     */
+    enum class DeviceType : int32_t {
```

Comment:
Should we have something that's the equivalent of 0?

Reply:
I'm thinking no as the device type must be something, it cannot be unspecified.

@philburk , could you also share your opinions here? Thanks,